### ISSUE TITLE
The OpenRec Server Terminal Open and closed

### ISSUE THREAD

**Issue by TarryTalvin** [recroom2016/OpenRec#5]

**Comment by Breelandthecat** [comment]
same

**Comment by RealMCoded** [comment]
Do you have dotNet 6.0 installed?

This version should work. I'm on mobile as I type this so I cannot verify. https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/runtime-desktop-6.0.33-windows-x64-installer

**Comment by Breelandthecat** [comment]
No, and I also got it fixed, I used .net 3.1.0 x64. Idk if it needs to be x64. Thank you though! Idk which one made it work but I installed both windows versions (not the server one) and it fixed it. The link to the version shows in the terminal when opening openrec with
The terminal and scroll down until you see 3.1.0 and get that. That I think is the right version.

**Comment by Breelandthecat** [comment]
probably the one for console apps.

**Comment by aqquad** [comment]
I'd recommend switching over to RebornRec instead of using OpenRec, which doesn't require you to install a .NET version and works right off the bat by opening the exe. Unless you really want a open source out-of-date server, RebornRec will do everything you want and better than OpenRec for old Rec Room, so if you want to get RebornRec, join us at discord.gg/yWBNpcAQTf and have fun!
